scripts/Muon/GUI/Common/plot_widget/plotting_canvas/plotting_canvas_view.py

```diff
@@ -207,24 +207,62 @@ def set_axis_ylimits(self, axis_number, ylims):
     def set_axes_limits(self, xlim, ylim):
         plt.setp(self.fig.axes, xlim=xlim, ylim=ylim)
 
+    def is_error_plotted(self):
+        #Function is defined in the presenter
+        pass
+
     def autoscale_y_axes(self):
```

Comment:
I feel like most of this should be in the presenter (and tested). This function should be short. 

@StephenSmith25 am I right in thinking we can get rid of the ADS call and use the context instead?

Reply:
Most of the autoscale function could be moved to the presenter but i am not sure how to use the context.

Reply:
https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_presenter.py#L466 I think is what you would want. @StephenSmith25 and @Mantid-Matthew what do you think?

Reply:
That looks like it would only get you the workspace name, not the workspace object which appears to be needed here. I feel like we should be able to get all the information from the plot, rather than having to grab the workspace - but haven't looked into this really.

Reply:
It is not possible to get the errors from the plot that is why i got it from the workspace only the x and y values can be gotten from the plot.